docs/stable/store/installation_with_rocm.md

```diff
@@ -2,8 +2,15 @@
 sidebar_position: 1
 ---
 
+
 # Installation with ROCm (Experimental)
 
+## Tested Hardware
```

Comment:
To ensure clarity, we should explicitly state the tested version of "SLLM" in the documentation or release notes. This can be manually updated each time we test SLLM on AMD GPUs to keep the information accurate and up-to-date.

Reply:
I will add a `Latest Tested Version` section